README.md

```diff
@@ -69,3 +69,53 @@ app.post('/contacts/:contactid', function (req, res) {
 	});
 });
 ```
+
+### Proposed item function reference
+Via the use of optional parameters and dedicated functions it is possible to retrieve anything from a single field on a single record to all fields on all records. Selection criteria and response timeouts may also be specified.
+
+The criteria parameter defines the query string to use to restrict the number of records that are returned. It followes the normal name=value&name=value... syntax
+
+The fields parameter defines which fields are to be output for each record. It expects a comma separated list of field names. Note that dataItemID, dataTypeID and lastUpdate will always be output.
+
+All returned JSON arrays and JSON objects are native javascript
+
+
+* Return all records of a type that match an optional criteria
+  + jsonArray = cmdb.getAllItems(locals, type, criteria = None, timeout = 6000)
```

Comment:
A people likely to want to specify different timeouts for individual requests?  Perhaps this should be specified in the config parameter when creating the CMDB object in the first place.